evap/static/css/bootstrap.less

```diff
@@ -10,7 +10,7 @@
 
 // Reset and dependencies
 @import "../bootstrap/less/normalize.less";
-@import "../bootstrap/less/print.less";
+//@import "../bootstrap/less/print.less"; // don't include bootstrap print css to enable printing colored results pages
```

Comment:
could you change this to `don't include this to enable ...`, i had difficulties parsing it :)